### ISSUE TITLE
[Request] Support Mac Catalyst

### ISSUE THREAD

**Issue by JoeySlomowitz** [Appboy/appboy-ios-sdk#230]
We really urgently need to see support for Braze on a Mac Catalyst app.  I've heard this is currently planned, but am hoping to see the priority elevated for this.

**Comment by Bucimis** [comment]
Hi @JoeySlomowitz,

Thanks for getting in touch. Catalyst support is in active development right now - I can't speak much more specifically to timelines but we're planning to release asap once support is ready.

**Comment by hokstuff** [comment]
Hi @JoeySlomowitz,

Just giving you an update on the Catalyst support - we've tried inserting a new architecture into our fat binary but then ran into an issue similar to [the one seen here](https://github.com/OneSignal/OneSignal-iOS-SDK/issues/636). Our new plan is to package our SDK into an xcframework that includes Catalyst support to distribute to clients, and that has just begun development.

Thanks!

**Comment by MihaiPantiru** [comment]
Hi @hokstuff,

Is is possible to share some updates on this one? I'm curios if it's doable to use your framework on Catalyst this year or at least to have the xcframework package. Thank you!

**Comment by hokstuff** [comment]
Hi @JoeySlomowitz and @MihaiPantiru,

Apologies for the delay regarding this project! It is currently in the works, and we expect to release the full feature by the end of the year. In the meantime, we have an unreleased version of the Core SDK (without UI code, assets, or dependencies) available as an XCFramework with Mac Catalyst support. If you would like to help test with it, please reach out to support@braze.com and we will be able to send it to you through that channel. Also, please link this thread in your email to avoid any confusion.

Thank you for your patience!

**Comment by EmperiorEric** [comment]
Any update on this? Will Braze be catching up to Catalyst soon?

**Comment by lowip** [comment]
Hi @EmperiorEric,

Sorry for the delay on this feature! We are currently in the final phases of our validation for Mac Catalyst support and we will likely be able to include this in our next release or the following one. We will update this thread soon!

**Comment by hokstuff** [comment]
Hi @JoeySlomowitz @MihaiPantiru @EmperiorEric,

We have released [version 3.32.0](https://github.com/Appboy/appboy-ios-sdk/releases/tag/3.32.0) which adds support for Mac Catalyst when integrating with Swift Package Manager. Please let us know if you have any issues - thank you!
